src/utils.js

```diff
@@ -1,14 +1,19 @@
-// flow
-export function findNextValid(steps, currentIndex) {
+// @flow
+import Step from './Step';
+
+export function findNextValid(steps: Array<typeof Step>, currentIndex: number) {
   const nextValid =
-    currentIndex + steps.slice(currentIndex + 1).findIndex(el => !el.autoSkip);
+    currentIndex + steps.slice(currentIndex + 1).findIndex(el => !el.autoSkip); // TODO
   return steps.length > nextValid ? nextValid + 1 : nextValid;
 }
 
-export function findPreviousValid(steps, currentIndex) {
+export function findPreviousValid(
+  steps: Array<typeof Step>,
+  currentIndex: number,
+) {
   const previousValid = [...steps]
     .reverse()
     .slice(steps.length - currentIndex)
-    .findIndex(el => !el.autoSkip);
+    .findIndex(el => !el.autoSkip); // TODO
```

Comment:
`findIndex` is a ES2015-method, so the client/consumer needs a polyfill to support ES5-browsers (similar to the URLSearchParams). That's ok in the case with otovoweb, but perhaps worth mentioning in README?

Reply:
Fixed in c12abbc